File: app/src/main/java/com/example/delivery/data/Favorites.java
package com.example.delivery.data;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class Favorites {
    @PrimaryKey
    private int entityId;
    public Favorites(int entityId){
        this.entityId=entityId;
    }

    public void setEntityId(int entityId){
        this.entityId=entityId;
    }
    public int getEntityId(){
        return entityId;
    }
}

File: app/src/main/java/com/example/deliverykotlin/data/Favorites.kt
package com.example.deliverykotlin.data

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
class Favorites{
        @PrimaryKey(autoGenerate = true)
        var id:Int?=null
        var entity:Int?=null
                set(value) {
                        field=value
                }

}